app/plugin.go

```diff
@@ -370,7 +374,17 @@ func (a *App) GetActivePluginManifests() ([]*model.Manifest, *model.AppError) {
 // activation if inactive anywhere in the cluster.
 // Notifies cluster peers through config change.
 func (a *App) EnablePlugin(id string) *model.AppError {
-	return a.ch.enablePlugin(id)
+	appErr := a.checkIfIntegrationMeetsFreemiumLimits(id)
+	if appErr != nil {
+		return appErr
+	}
```

Comment:
Should `checkIfIntegrationMeetsFreemiumLimits` also be checked when a plugin get installed? How does the UX look like if an admin installs a plugin via the Marketplace that would push the server over the limit?

Reply:
@mickmister - is the current design that we notify if new plugins/apps get installed and we are over the limit? I thought that we only do that when we enable installed plugins - user can install as many as they please.